Ingress-how-to.md

```diff
@@ -0,0 +1,83 @@
+# Ingress How To
+
+This document explains who to configure the OpenShift ingress controller to take advantage of the keepalived-operator.
+
+## Ingress configuration
+
+Assuming you have a properly installed keeapalived configuration, proceed as follows:
+
+Create an ingress controller:
+
+```yaml
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: my-keepalived-ingress
+  namespace: openshift-ingress-operator
+spec:
+  domain: myingress.mydomain
+  replicas: 2
+  endpointPublishingStrategy: 
+    type: Private
+```
+
+You can add any other field needed to your configuration, the important thing here is `endpointPublishingStrategy: Private`.
```

Comment:
Can you please point out why that is the case and from where you got this information?

i cannot find info about it: https://docs.openshift.com/container-platform/4.6/rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.html